stream_alert_cli_module/terraform_generate.py

```diff
@@ -331,7 +331,8 @@ def generate_kinesis(cluster_name, cluster_dict, config):
     """
     logging_bucket = '{}.streamalert.s3-logging'.format(
         config['global']['account']['prefix'])
-    firehose_suffix = config['clusters'][cluster_name]['modules']['kinesis']['firehose']['s3_bucket_suffix']
+    firehose_suffix = config['clusters'][cluster_name]['modules']['kinesis']['firehose'][
```

Comment:
The standard is to never use `\`; the only such cases were these long dict accesses and I switched all of them to this new format